### PR TITLE
Content - Designers onboarding

### LINKED ISSUE #1296: Getting Started: Designers steps not complete
## Detailed description

> The instructions describe two ways to install the carbon sketch kit, and one way to install the IDL design Kit, without describing the dependency between the kits or clearly stating the way you _should_ do it (the way that will be updated - Sketchcloud) 

> https://next.carbondesignsystem.com/getting-started/designers

> At the conclusion designers should have all of the kits required to start working. Designers are not getting or understanding the requirement for a separate IDL kit (as reported in Slack https://ibm-cloudplatform.slack.com/archives/C0M053VPT/p1552424212177600)

> browser: Chrome, but any

> version: 10 (next.carbondesignsystem.com)

## Steps to reproduce the issue

1. Navigate to https://next.carbondesignsystem.com/getting-started/designers
2. Follow the instructions; assuming they complete the first instruction (https://next.carbondesignsystem.com/getting-started/designers#sketch-shared-library-(preferred)) they are unlikely to continue to the second since they are presented as alternatives (https://next.carbondesignsystem.com/getting-started/designers#get-the-kit-via-box-(ibm-internal-only))

## Additional information
Expected: Instructions capture our clearly preferred install method, and make clear that there are two required kit installs. Retain a pointer to the containing repo if users want to inspect it - in understanding how to make a symbol for submission for example - but make clear that that file will be orphaned and never part of the automatic update regime

1: Install the IDL Kit, which includes fonts, colors, icons, and other resources shared across all embodiments of the IBM Design Language (card points to IDL in Sketchcloud now, `carbon-elements` kit when available)
2: Install the Carbon v10 kit(s) via Sketch cloud
3: details about setup in Sketch

### PR DESCRIPTION
**Pull request by connor-leech**
Closes #1296

Redesigned the onboarding experience for designers from the ground up. The focus here was lightweight, action oriented structure with information that serves all visitors to our sight. I sought to remove any internal guidance, embracing Carbon's mission to be truly open-source.

Our onboarding process now requires minimal navigation and is entirely cloud based. Anybody, anywhere can begin building with Carbon in moments.

#### Changelog

**New**

- A completely redesigned Getting started experience.
- Guidance for Sketch libraries, allowing Carbon to update across its userbase in real-time
- Basic, action-oriented walk through of the kit's anatomy.
- Up-to-date images reflecting the current kit and process

### CODE REVIEW

File: src/content/getting-started/designers/designers.mdx [hunk 1]
```diff
@@ -92,12 +92,14 @@ You can see a full breakdown of the 2x Grid at [IBM.com/design](https://www.ibm.
...
-![Carbon symbol menu navigation](/images/symbol-nav.png)
+![Carbon symbol menu navigation](/images/IDL-kit-library.png)
 
 </ImageComponent>
 
+
+
 There are two kinds of symbols — `library symbols` and `document symbols`. Library symbols are available in any Sketch document, while document symbols are specific to the document in which they are found.
 
 From the main menu select `Insert → Symbols → Carbon Design System` to add symbols onto your page.
```
Comment (chrisconnors-ibm) [inline src/content/getting-started/designers/designers.mdx]: should be "From the `Insert` menu, select `Symbols → Carbon Design System` to add symbols onto your page.

Comment (chrisconnors-ibm) [reply src/content/getting-started/designers/designers.mdx]: @aagonzales is this were we should say something about 'don't disconnect Symbols bc now they are so much more flexible yada yada yada'?

Comment (connor-leech) [reply src/content/getting-started/designers/designers.mdx]: @chrisconnors-ibm Thanks for looking at this. In the same vein as this Q for Anna, is my language correct regarding layer styles? 

"If you elect to override a preset layer style, use the existing override styles found in the right-hand panel."

Comment (aagonzales) [reply src/content/getting-started/designers/designers.mdx]: People shouldn't be over-riding layer styles or breaking symbols. Once we have the other themes in there we can say something about how you should switch to the correct theme symbols and not over-ride layer styles. We should emphasis not detaching form symbols. 

Somewhere we can do a section on editing the sketch file to update to your own custom theme but out of the box IBMers should not be messing with layer overrides. In fact, we've actually locked a lot of those layers and don't even allow users to edit them. The would have to unlock it on the master to edit.

**Comment by aagonzales** [comment]
This workshop this week made me think that maybe we should also have a section on our intent to expand the toolset. Or what you can do if you don't use Sketch (like the workshop people only us axure).

**Comment by connor-leech** [comment]
@aagonzales I think you're probably right, but I don't think including intent to expand the toolset should be on the Getting Started page.

I feel like we should keep this page only as actionable, source of truth information to avoid any confusion even if it's not 100% inclusive of all circumstances.

I'd agree, however, that we should mention in our roadmap what, if any, tooling is on the way for designers.

File: src/content/getting-started/designers/designers.mdx (hunk 1 above)
Comment (connor-leech) [reply src/content/getting-started/designers/designers.mdx]: I misunderstood how the layer styles were categorized and corrected the language.

**Comment by claycrenshaw** [comment]
Connor, I put my consolidated feedback here: https://ibm.box.com/s/njrpeteqik48qofqc43xqu13kufjx310 Please take a look and let me know if you have any questions!

File: src/content/getting-started/designers/designers.mdx [hunk 2]
```diff
@@ -2,120 +2,179 @@
...
-</ImageComponent>
+#### Structure
+
+At the core of the 2x Grid concept is the idea of divisions of two. This can be applied to most surfaces, 3-D objects, and architecture as a means to divide space into a helpful grid system. 
 
-### Download the kit from GitHub
+Dividing your space into 2, 4, 8, 16, 32, or 64 columns gives you the basis of the system with the purpose of helping you make decisions and organize your content.
 
-This is the best method for one-time downloads of the kit, or if you are looking for previous versions.
+You can see a full breakdown of the 2x Grid at [IBM.com/design](https://www.ibm.com/design/language/elements/2x-grid).
```
Comment (aagonzales) [inline src/content/getting-started/designers/designers.mdx]: Should we be linking out to the IDL grid page or our grid page (included in [layout](https://next.carbondesignsystem.com/guidelines/layout/#2x-grid-fundamentals))? Just an open question

File: src/content/getting-started/designers/designers.mdx [hunk 3]
```diff
@@ -2,120 +2,179 @@
...
-2. Once you have installed and gained access to the `IBM Design Kit` folder, navigate to the `product design` folder. Select the desired version of Carbon.
-3. When updates are available to the kit library, a purple `Library Updates Available` badge will appear in the top right corner of your Sketch file.
-4. Click `Library Updates Available` then `Accept Updates` in the modal.
+![2x Grid overview](/images/grid-1.png)
 
-<ImageComponent cols="8" caption="Update indicator">
+</ImageComponent>
 
-![Update indicator screenshot](images/designers-5.png)
+All designs should start with the 2x Grid template. Once you have successfully [saved the grid](#gat-the-kit) as a template, navigate to `File → New file from Template` and select the grid template.
```
Comment (aagonzales) [inline src/content/getting-started/designers/designers.mdx]: We can maybe add in something about talk about toggling on and off the grid and layout (maybe add a gif). Its with the control (control+g) and (control+L)
Mar-19-2019 17-16-53

File: src/content/getting-started/designers/designers.mdx [hunk 4]
```diff
@@ -2,120 +2,179 @@
...
+Carbon symbols are built to be flexible, and designers should not detach symbols from the library. Once a symbol is detached, you will no longer receive updates as they are released.
+
+#### Structure
 
-![Kit file screenshot](images/designers-3.png)
+Symbols are organized by component; after selecting a component, you'll see the variations, pieces and states of that component. 
+
+- To add the whole component, simply locate the variant you want and drag it into the work space.
+
+- For even more configurablility, use the individual building blocks under "Pieces." Be sure your custom design adheres to our [design guidelines](/guidelines/accessibility/overview).
```
Comment (aagonzales) [inline src/content/getting-started/designers/designers.mdx]: The building blocks are under "items" not "pieces." Pieces are like tiny micro parts of a component symbol. These are the best [definitions](https://github.ibm.com/duo/design-kit/wiki/Terminology) but they might help clear it up for you a bit.

File: src/content/getting-started/designers/designers.mdx (hunk 2 above)
Comment (connor-leech) [reply src/content/getting-started/designers/designers.mdx]: Layout page makes more sense, we don't want people to unnecessarily leave our site.

File: src/content/getting-started/designers/designers.mdx (hunk 3 above)
Comment (connor-leech) [reply src/content/getting-started/designers/designers.mdx]: I added this! Thank you.

File: src/content/getting-started/designers/designers.mdx (hunk 4 above)
Comment (connor-leech) [reply src/content/getting-started/designers/designers.mdx]: Changed this too!

**Pull request merged**